src/test/java/com/google/sps/TextAnalyserTest.java

```diff
@@ -0,0 +1,132 @@
+package com.google.sps.data;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.junit.Test;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+
+@RunWith(JUnit4.class)
+public final class TextAnalyserTest {
+
+  @Test
+  public void analysePositiveSentiment() {
+    try {
+      TextAnalyser textAnalyser = new TextAnalyser("Today is a special day. It is your birthday, so enjoy it!");
+      assertTrue("Sentiment score is not positive.", textAnalyser.getSentimentScore() > 0.0F);
+    } catch (IOException e) {
+      fail(e.getMessage());
+    }
+  }
+
+  @Test
+  public void analyseNegativeSentiment() {
+    try {
+      TextAnalyser textAnalyser = new TextAnalyser("I don't like this.");
+      assertTrue("Sentiment score is not negative.", textAnalyser.getSentimentScore() < 0.0F);
+    } catch (IOException e) {
+      fail(e.getMessage());
+    }
+  }
+
+  @Test
+  public void analysePositiveMood() throws IOException {
+    TextAnalyser textAnalyser = new TextAnalyser("Happy Birthday!");
+    String expected = "thrilled";
+    String actual = textAnalyser.getMood();
+
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  public void analyseNegativeMood() throws IOException {
+    TextAnalyser textAnalyser = new TextAnalyser("Smoking, well, there is no healthy amount of smoking." +
+                                                  "Healthy here meant never having smoked.");
+    String expected = "depressed";
+    String actual = textAnalyser.getMood();
+
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  public void testForCorrectEvents() {
```

Comment:
the `test` prefix is redundant for @Test annotated methods

be consistent throughout at least throughout this file, ideally consistent across the project

there is little guidance in the style guide: https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names